pywb/static/query.js

```diff
@@ -65,26 +65,7 @@ function RenderCalendar(init) {
     '=!=': 'Is Not',
     '=!~': 'Does Not Begins With'
   };
-  this.text = {
```

Comment:
Documentation (i8n, static files)

Documentation requirement for all static files affected by i8n.